src/device/config.h.in

```diff
@@ -19,7 +19,7 @@
 /*----------------------------------------------------------------------------*/
 /* Mark functions for compilation on host and device */
 
-#if defined HAVE_CUDA && defined USE_CUDA
```

Comment:
Did you universally remove `HAVE_CUDA` from the codebase?  Did you check the `config/` subdirectory also?

Reply:
No, I haven't removed it. I think it still makes sense inside of CMake. The way I see it, CUDA's availability seems like a configure time step, using CUDA should only be true if `HAVE_CUDA` is true and should only matter at compile time.